tests/integration/status-list-credential-module.test.js

```diff
@@ -91,149 +102,152 @@ buildTest('StatusListCredential Module', () => {
-      ),
+        "Can't unsuspend indices for credential with `statusPurpose` = `revocation`, it's only possible with `statusPurpose` = `suspension`"
+      )
     );
     expect(
-      cred.update(ownerKey, { unsuspendIndices: revokeIds }),
+      cred.update(ownerKey, { unsuspendIndices: revokeIds })
     ).rejects.toEqual(
       new Error(
-        "Can't unsuspend indices for credential with `statusPurpose` = `revocation`, it's only possible with `statusPurpose` = `suspension`",
-      ),
+        "Can't unsuspend indices for credential with `statusPurpose` = `revocation`, it's only possible with `statusPurpose` = `suspension`"
+      )
     );
     expect(await cred.revokedBatch(revokeIds)).toEqual(
-      Array.from(revokeIds, () => true),
+      Array.from(revokeIds, () => true)
     );
   }, 40000);
 
-  test('Can unsuspend from a status list credential', async () => {
+  test("Can unsuspend from a status list credential", async () => {
     const credential = await StatusList2021Credential.create(
       ownerKey,
       statusListCredId,
-      { statusPurpose: 'suspension', revokeIndices: revokeIds },
-    );
-    let [update, sig, nonce] = await dock.statusListCredential.createSignedUpdateStatusListCredential(
-      statusListCredId,
-      credential,
-      ownerDID,
-      pair,
-      1,
-      { didModule: dock.did },
+      { statusPurpose: "suspension", revokeIndices: revokeIds }
     );
+    let [update, sig, nonce] =
+      await dock.statusListCredential.createSignedUpdateStatusListCredential(
+        statusListCredId,
+        credential,
+        ownerDID,
+        pair,
+        { didModule: dock.did }
+      );
     await dock.statusListCredential.updateStatusListCredential(
       update,
-      [[sig, nonce]],
-      false,
-    );
-    let fetchedCred = await dock.statusListCredential.fetchStatusList2021Credential(
-      statusListCredId,
+      [[nonce, sig]],
+      false
     );
+    let fetchedCred =
+      await dock.statusListCredential.fetchStatusList2021Credential(
+        statusListCredId
+      );
     expectEqualCreds(credential, fetchedCred);
     await fetchedCred.update(ownerKey, { unsuspendIndices: revokeIds });
     expect(await fetchedCred.revokedBatch(revokeIds)).toEqual(
-      Array.from(revokeIds, () => false),
-    );
-    [update, sig, nonce] = await dock.statusListCredential.createSignedUpdateStatusListCredential(
-      statusListCredId,
-      fetchedCred,
-      ownerDID,
-      pair,
-      1,
-      { didModule: dock.did },
+      Array.from(revokeIds, () => false)
     );
+    [update, sig, nonce] =
+      await dock.statusListCredential.createSignedUpdateStatusListCredential(
+        statusListCredId,
+        fetchedCred,
+        ownerDID,
+        pair,
+        { didModule: dock.did }
+      );
     await dock.statusListCredential.updateStatusListCredential(
       update,
-      [[sig, nonce]],
-      false,
+      [[nonce, sig]],
```

Comment:
Why change the order and introduce another breaking change? Better to keep the serialization of DIDSig the same on the node.